index.html

```diff
@@ -609,8 +639,24 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             </li>
           </ol>
         </li>
-        <li>If the <var>add to performance entry buffer</var> flag is set, add
-        <var>new entry</var> to the <a>performance entry buffer</a>.
+        <li>
+          Determine if you should add <var>new entry</var> to a 
```

Comment:
This seems like something you'd add as a code comment. In a spec, there are a couple of ways to do that: a) Define it as an independent algorithm which name states its use (kinda like helper functions with a meaningful name) b) Add it as a note

Reply:
So it is more like the previous statement when I used the "get the releveant performance entry tuple" algorithm.  Let me know if you think it still makes more sense as a comment.  To me it is a pretty central part of queuing a PerformanceEntry object.